Bottom: 1c0d2f49091c0c9ebf1e7eadf40bc7094d0978de
Top:    04cc79d4b12db0b5a80658b1d33cf9eded379171
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-28 21:30:46 +0100

Refresh of parse_y-y-don-t-bother-setting

---

diff --git a/src/parse_y.y b/src/parse_y.y
index ed4d183..c6f5246 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -220,7 +220,7 @@ parsepcb
 		    yyData->pcb = yyPCB;
 		    yyData->LayerN = 0;
 		    layer_group_string = NULL;
-		}
+		  }
 		  element
 		  {
 		    PCBType *pcb_save = PCB;
